src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Security/Helpers/InsecureObjectGraphResult.cs

```diff
@@ -87,7 +86,7 @@ public string GetDisplayString()
             else if (this.InsecureAttributeTypedConstant != null)
             {
                 TypedConstant t = (TypedConstant)this.InsecureAttributeTypedConstant!;
-                return t.ToCSharpString();
+                return t.ToString(); // TODO: What to do here
```

Comment:
@dotpaul May I have some help to refactor this bit?

Reply:
I'll take a look later this week. Last time I looked there wasn't a nice language agnostic solution, which is why it's using ToCSharpString(). Maybe we can get to the syntax node for the attribute's constructor argument somehow.

Reply:
Yep the problem is just that it is accessing C# specific from the common layer.